src/index.html

```diff
@@ -17,6 +17,43 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1></h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="#">Samsung</a></li>
+          <li><a href="#">Smartphones</a></li>
+          <li>
+            <a
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li><a href="#">Gadgets</a></li>
+          <li><a href="#">Tablets</a></li>
+          <li><a href="#">Photo</a></li>
+          <li><a href="#">Video</a></li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML for better readability, but don't add them between parent and child elements.